gke-usage-metering/src/gkeUsageMetering.ts

```diff
@@ -155,7 +158,8 @@ namespace gkeUsageMetering {
         .setText('The GCP billing table ID is not specified')
         .throwException();
     }
-    if ((fullBillingTable.match(/\./g) || []).length != 2) {
+    // Bigquery table IDs can contain letters, numbers and underscores
```

Comment:
Similarly, let's mention that `fullBillingTable` should be in the format of `${PROJECT_ID}.${DATASET_ID}.${TABLE_ID}`.

And make sure to update the regex for project ID.

Reply:
Done